cmd/kube-rbac-proxy/app/kube-rbac-proxy.go

```diff
@@ -207,6 +198,7 @@ func Run(opts *completedProxyRunOptions) error {
 	handler := identityheaders.WithAuthHeaders(proxy, cfg.KubeRBACProxyInfo.UpstreamHeaders)
 	handler = kubefilters.WithAuthorization(handler, authz, scheme.Codecs)
 	handler = kubefilters.WithAuthentication(handler, authenticator, http.HandlerFunc(filters.UnauthorizedHandler), cfg.DelegatingAuthentication.APIAudiences)
+	// passing an empty RequestInfoFactory results in attaching a non-resource RequestInfo to the context
```

Comment:
@ibihim I'm not sure if I'm missing any other implication of passing an empty `RequestInfoFactory`, please advise :)